### PR TITLE
Revert Material 3, Dynamic Colors Option, and Bottom Project Settings

### PR DESCRIPTION
**Pull request by trindadedev13**

### CODE REVIEW

**Comment by aikrq** [comment]
I think it's not necessary to revert bottom sheet project options.

**Comment by trindadedev13** [comment]
> I think it's not necessary to revert bottom sheet project options.

donee

File: app/src/main/res/layout/dialog_project_settings.xml [hunk 1]
```diff
@@ -68,13 +69,13 @@
...
             </com.google.android.material.textfield.TextInputLayout>
 
+
             <com.google.android.material.card.MaterialCardView
                 style="?attr/materialCardViewFilledStyle"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="10dp"
-                app:cardBackgroundColor="?attr/colorSurfaceContainer"
-                app:strokeWidth="0.5dp">
```
Comment (aikrq) [inline app/src/main/res/layout/dialog_project_settings.xml:77]: Why did you remove it?

File: app/src/main/res/layout/item_xml_command.xml [hunk 2]
```diff
@@ -9,8 +9,7 @@
     android:layout_marginTop="10dp"
     android:focusable="true"
     android:clickable="true"
-    app:cardBackgroundColor="?attr/colorSurfaceContainer"
-    app:strokeWidth="0.5dp">
```
Comment (aikrq) [inline app/src/main/res/layout/item_xml_command.xml:13]: This too.

File: app/src/main/res/layout/export_project.xml [hunk 3]
```diff
@@ -258,8 +256,7 @@
                 android:layout_margin="8dp"
                 app:cardBackgroundColor="?attr/colorSurfaceContainer"
                 app:cardElevation="0dp"
-                app:contentPadding="14dp"
-                app:strokeWidth="0.5dp">
```
Comment (aikrq) [inline app/src/main/res/layout/export_project.xml:262]: This too.

File: app/src/main/res/layout/dialog_project_settings.xml [hunk 4]
```diff
@@ -102,13 +103,13 @@
...
             </com.google.android.material.card.MaterialCardView>
 
+
             <com.google.android.material.card.MaterialCardView
                 style="?attr/materialCardViewFilledStyle"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="20dp"
-                app:cardBackgroundColor="?attr/colorSurfaceContainer"
-                app:strokeWidth="0.5dp">
```
Comment (aikrq) [inline app/src/main/res/layout/dialog_project_settings.xml:111]: This too.

File: app/src/main/res/layout/dialog_project_settings.xml [hunk 5]
```diff
@@ -141,8 +143,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginTop="10dp"
-                app:cardBackgroundColor="?attr/colorSurfaceContainer"
-                app:strokeWidth="0.5dp">
```
Comment (aikrq) [inline app/src/main/res/layout/dialog_project_settings.xml:145]: This too.

**Review by aikrq — CHANGES_REQUESTED** [review]

**Comment by trindadedev13** [comment]
@aikrq in response to all the "reviews":
I reverted all the commits that had to do with ProjectSettings, and in those commits, I had postponed these Attributes both in the ProjectSettings Layout and in the revised files

**Comment by aikrq** [comment]
You can add them manually.

**Comment by trindadedev13** [comment]
done

**Comment by waelchateur** [comment]
why ?? i am planing of pushing my changes for m3 and those feature are necessairy to use material components

**Comment by trindadedev13** [comment]
> why ?? i am planing to push my changes for m3 and those feature are necessairy to use material components

idk, for me this works fine, but the other devs said that this breaks old projects

**Comment by waelchateur** [comment]
> > why ?? i am planing to push my changes for m3 and those feature are necessairy to use material components
> 
> idk, for me this works fine, but the other devs said that this breaks old projects

if those are turned off by default how they can affect old projects already tested a lot of projects and they work fine more many want m3 so here we r giving them m3 now they can make beautiful apps more you can disable them for minsdk21 like what i did cause no more future updates needed for old phones maybe only fixing some libs like firebase or onsignal or kotlin exept that your feature is very needed

**Comment by waelchateur** [comment]
> > why ?? i am planing to push my changes for m3 and those feature are necessairy to use material components
> 
> idk, for me this works fine, but the other devs said that this breaks old projects

how about keeping enable_bridgeless_themes and if use material3 active it will be disabled and m3 theme injected instead

**Comment by aikrq** [comment]
Why there's so many force pushes?

**Comment by trindadedev13** [comment]
> Why there's so many force pushes?

what problem?

**Review by aikrq — APPROVED** [review]

**Comment by RoyTheReal** [comment]
No, PLEASE, don't get rid of the new material 3 + dynamic colors toggle, it works amazing! I was so happy to see that it was added. Way easier than adding it manually. Keep it in and just add a warning it _may_ break older projects. This feature is literally my favorite feature currently.

**Comment by RoyTheReal** [comment]
BRING THE FEATURES BACK PLEASE! They are literally disabled on default. I tested the feature and it worked on ALL of my old projects.

**Comment by RoyTheReal** [comment]
@trindadedev13 Dude revert this revert.

**Comment by Ohyeah66h** [comment]
Yeah bring this feature back, literally just add a warning. Manually doing it never works correctly. The toggle is a huge quality of life enhancement

**Comment by hoxavel457** [comment]
I vote to bring this feature back as well. I love this feature.

**Comment by RoyTheReal** [comment]
@trindadedev13 @aikrq A lot of people really want these features back, including me. Please, add them back. You guys are awesome :)

**Comment by Ohyeah66h** [comment]
True :)

**Comment by RoyTheReal** [comment]
@trindadedev13 Please bring the options back, I was using the feature

**Pull request merged**